README.md

```diff
@@ -33,21 +33,21 @@ The extension also adds a Java DSL that works with the Java classes generated by
 
 Add the extension JAR dependency to your project:
 
-=== "Gradle"
+#### Gradle
```

Comment:
```suggestion
Gradle:
```
to more accurately match what's currently on https://wiremock.org/docs/grpc/

Reply:
good catch! thanks :)